src/components.js

```diff
@@ -57,6 +73,22 @@ export class PieceView extends React.Component {
   }
 }
 
+export class NextPieceView extends React.Component {
```

Comment:
I would suggest placing each of these Components in their own files, properly organized in your project's file structure.